README.md

```diff
@@ -113,6 +115,70 @@ const blogs = await mddb.getFiles({
 });
 ```
 
+# Configuring `mddb.config.js`
```

Comment:
What happened to our simple code config in the first version. I'd do that first ...

Overall this approach seems pretty sound and we would need it at some point. I'd just like to get the details right with some code before having to do config stuff.